jdbc/src/main/java/org/apache/zeppelin/jdbc/SqlCompleter.java

```diff
@@ -230,8 +230,7 @@ public static Set<String> getSqlKeywordsCompletions(DatabaseMetaData meta) throw
           keywords += "," + driverKeywords.toUpperCase();
         }
       } catch (Exception e) {
```

Comment:
We should catch the explicit exceptions. Also, `SQL-Exception` should be removed as a throw declaration.
```suggestion
      } catch (IOException | SQLException e) {
```